test/e2e/case5_policy_automation_test.go

```diff
@@ -140,6 +140,17 @@ var _ = Describe("Test policy automation", func() {
 
 				return len(ansiblejobList.Items)
 			}, 30, 1).Should(Equal(1))
+			By("Job TTL should match default (1 day)")
+			Eventually(func() interface{} {
+				ansiblejobList, err := clientHubDynamic.Resource(gvrAnsibleJob).Namespace(testNamespace).List(
+					context.TODO(), metav1.ListOptions{},
+				)
+				Expect(err).To(BeNil())
+				_, err = utils.KubectlWithOutput("get", "ansiblejobs", "-n", testNamespace)
```

Comment:
What's the purpose of this when you made the API call above using the Go client library?